src/main/kotlin/com/rooplor/classcraftbackend/configuration/FirebaseConfig.kt

```diff
@@ -0,0 +1,30 @@
+package com.rooplor.classcraftbackend.configuration
+
+import com.google.auth.oauth2.GoogleCredentials
+import com.google.firebase.FirebaseApp
+import com.google.firebase.FirebaseOptions
+import com.rooplor.classcraftbackend.messages.ErrorMessages
+import com.rooplor.classcraftbackend.services.logger.LoggerService
+import org.springframework.context.annotation.Configuration
+import java.io.FileInputStream
+
+@Configuration
+class FirebaseConfig(
+    private val loggerService: LoggerService,
+) {
+    init {
+        try {
+            val serviceAccount = FileInputStream("src/main/resources/serviceAccountKey.json")
```

Comment:
Can we use path as configuration variable?

Reply:
WDYM? Create a path as variable right? or do you mean something else?